stack.yaml

```diff
@@ -1,6 +1,6 @@
-resolver: lts-12.9
+resolver: nightly-2018-10-01
```

Comment:
I guess you shouldn't change the `stack.yaml` file as we still want to support `8.4.3` instead you can put this into `stack-8.6.1.yaml` file 🙂